changelogs/fragments/colleciton_flex_ac_dir_paths.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - Be smarter about colleciton paths ending with ansible_collections, emulating a-galaxy behaviour. Issue 72628
```

Comment:
s/colleciton/collection/

And for consistency `(https://github.com/ansible/ansible/issues/72055)`